packages/engine/src/scene/components/VariantComponent.tsx

```diff
@@ -123,8 +124,18 @@ export const VariantComponent = defineComponent({
   reactor: () => {
     const entity = useEntityContext()
     const variantComponent = useComponent(entity, VariantComponent)
-
     const instancingComponent = useOptionalComponent(entity, InstancingComponent)
+    let childEntity = getChildrenWithComponents(entity, [GLTFComponent]).at(0)
```

Comment:
this should be a useHookstate storing the child entity, set when it is created, rather than calculating it each time react rerenders. `let` is essentially an anti-pattern in react